### ISSUE TITLE
Unknown error occured for new release

### ISSUE THREAD

**Issue by MaxNoe** [zenodo/zenodo#1907]
Hi,

For our latest release v1.1.3 of https://github.com/fact-project/fact-tools, the zenodo deploy failed with "Unknown error occured".

We did not change anything regarding the zenodo config.

https://zenodo.org/account/settings/github/repository/fact-project/fact-tools#

Any help would be highly appreciated!

**Comment by LaboMaranger** [comment]
Hi,
I have the same problem.

**Comment by kazunarikudo** [comment]
Hi,

I confirmed Zendo successfully hooked our two trial releases and I could publish them as well yesterday (November 21st, 2019). However, it still hooks releases but always gives me the following error message today (November 22nd, 2019):

> "errors": "Unknown error occured." [sic]

What helpful information, it is. :-) The message can be seen at `https://zenodo.org/account/settings/github/`.

As far as hear the report from @LaboMaranger , perhaps there could be a link between us; he left the comment at around 5:00 a.m. on 22nd, in my time zone. I noticed the error this morning and it's not yet resolved as of 10:00 p.m on 22nd.

Another observation I've had is I can successfully publish the hooked released from Zendo Sandbox.

I would like to publish the latest releases so any suggestions or updated will be appreciated.

Regards,
Kazunari

**Comment by LaboMaranger** [comment]
Hi,

I tried on sandbox.zenodo.org and it worked.
I'm not sure though that it was only a test upload since it still attributed a DOI (10.5072/zenodo.420967).
I hope it is only a test upload since there is no history of previous versions.

**Comment by kazunarikudo** [comment]
Hi, does yours still fail on the real Zenodo?

**Comment by LaboMaranger** [comment]
@kazunarikudo I tried about an hour ago and it did not worked.

**Comment by kazunarikudo** [comment]
@LaboMaranger Are you new to Zenodo? Or have enough experience with that?

**Comment by LaboMaranger** [comment]
@kazunarikudo I've used Zenodo a few times before, but all went well each time. The webhooks via github worked the previous times. I have no idea what's going on.

**Comment by kazunarikudo** [comment]
@LaboMaranger Maybe we are in the same situation. Zenodo worked for me yesterday but is not working now. Probably just waiting for the Zenodo team to come would be a better idea. I do not think the issue can be resolved not having help from them. Nothing is obvious from our side.

**Comment by kazunarikudo** [comment]
@LaboMaranger I’ll give you an update every time I made a try. Could you also give me an update when you can, please? Having faced this issue, I feel comparison of the result on each end can be valuable and definitely minimize the number of trials we will have.

**Comment by LaboMaranger** [comment]
@kazunarikudo Yes I will give you an update when I try a new release, or if Zenodo solves the problem. I agree that we should wait for external inputs to solve this problem.

**Comment by kazunarikudo** [comment]
Hooks a release but gives me the same error. No difference. No progress.
```
Time zone in Tokyo (GMT+9)
Friday, November 22, 2019, 9:36 AM
```

**Comment by kazunarikudo** [comment]
Still hooks a release but gives me the same error. No difference. No progress.
```
Time zone in Tokyo (GMT+9)
Monday, November 25, 2019, 10:05 AM
```

**Comment by LaboMaranger** [comment]
@kazunarikudo Hi, I did not try before today, and now it is working fine.

**Comment by kazunarikudo** [comment]
@LaboMaranger Thank you very much for your hard working. Yes, I've confirmed it's working!

**Comment by Glignos** [comment]
Closing since the issue has been resolved.

**Comment by MaxNoe** [comment]
@Glignos please reopen. This happened several times and there was never a reason found and the error message is utterly unhelpful.

**Comment by slint** [comment]
@MaxNoe, the reason we closed the issues, is because Zenodo GitHub release problems are inherently too diverse in scope to be handled here on a per-user basis. Since we usually require Zenodo user account information and further (sometimes private) context regarding e.g. GitHub organization/repo ownership and permissions to solve this, we prefer to address these via our support line at https://zenodo.org/support.

On a side note, we've identified and clustered some of these problems, and plan to provide more helpful messages, although the issue still remains that to fix these users have to go through our support channel. We're very sorry for the inconvenience this causes
